### ISSUE TITLE
How do I handle API Base URLs for Resources?

### ISSUE THREAD

**Issue by melkristian** [starlight36/fetch-http-client#12]
What if my API resource endpoint looks like below? How do I instantiate a Client?
```
/api/myapi/resourceA
/api/myapi/resourceB
```
In this example, my actual base URL is `<host>/api/myapi/` but initializing a client seems to only recognize up to `<host>/api/`.

**Comment by yedaodao** [comment]
@melkristian Thank for your advertisement. We will add this feature today.

**Comment by jinyang1994** [comment]
@melkristian You might have used it this way.

```javascript
import FetchHttpClient from 'fetch-http-client';

const client = new FetchHttpClient('/api/myapi/');

client.get('/resourceA');
```

But it should be used like this.

```javascript
client.get('resourceA');
```

**Comment by melkristian** [comment]
@jinyang1994 You are correct! This does work.

Also, I just realized that if I initialized at the resource level like:
```js
const client = new FetchHttpClient('/api/myapi/resourceA');
```

The most basic get call would look like:
```js
client.get(''); // with an empty string, works
client.get(); // errors out
client.get('/'); // makes a call to /api/, incorrect
```

Are the failing issues above expected?

The scenario that I'm trying to avoid is having to constantly write out the resource. For now, I can close the issue since there is away to achieve my original intention for using fetch-http-client. But using the convenience methods like `post('')` and `get('')` seem a bit awkward.

Thanks a lot! :)
